playbooks/robusta_playbooks/job_restart_on_oomkilled_community.py

```
@@ -41,7 +41,7 @@ def job_restart_on_oomkilled_community(event: JobEvent, params: IncreaseResource
     try:
         pod = get_job_latest_pod(job_event)
 
-    except:
+    except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)